docs/pages/web/sheet.js

```diff
@@ -261,6 +273,221 @@ Sheet comes in 3 sizes: small (\`sm\`), medium (\`md\`), and large (\`lg\`).
             }
           />
         </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Dismiss confirmation"
+          description={`There are two ways Sheet can be dismissed: internally-controlled and externally-controlled dismiss actions.
+
+The three internally-controlled or component-controlled dismiss actions are:
+- when the \`ESC\` key is pressed
+- when the backdrop is clicked
+- when the dismiss IconButtons are clicked
```

Comment:
There are multiple dismiss icon buttons?

Reply:
In my mind yes, for the user no. In the code, there are 2 haha

Reply:
DONE